tests/unit/records/test_records_serializers_fields_export.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from inspirehep.modules.records.serializers.fields_export import (
+    get_authors_with_role,
+    get_author,
+    bibtex_document_type,
+    make_author_list,
+    MAX_AUTHORS_BEFORE_ET_AL,
+    bibtex_type_and_fields,
+    get_year,
+    get_publication_info,
+    get_journal,
+    get_volume,
+    get_slac_citation,
+    get_report_number,
+)
+
+test_record = {
```

Comment:
you want this to be a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html).